backend/endpoints/rooms.py

```diff
@@ -61,3 +63,80 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+@router.delete("/org/{org_id}/members/{member_id}/rooms/{room_id}",
```

Comment:
Just for consistency, i think this should be org/org_id/rooms/room_id/members/member_id